packages/module/patternfly-docs/content/examples/PropertiesSidePanel.md

```diff
@@ -26,44 +26,42 @@ import OkIcon from '@patternfly/react-icons/dist/esm/icons/ok-icon';
 import ExternalLinkAltIcon from '@patternfly/react-icons/dist/esm/icons/external-link-alt-icon';
 import GlobeIcon from '@patternfly/react-icons/dist/esm/icons/globe-icon';
 
-<div style={{ display: 'inline-block', padding: '15px', border: '1px solid grey' }}>
-  <PropertiesSidePanel>
-    <PropertyItem label="Operator Version" value="0.9.8 (latest)" />
-    <PropertyItem
-      label="Certified Level"
-      value={
-        <span>
-          <OkIcon style={{color: '--pf-t--global--color--status--success--default'}} /> Certified
-        </span>
-      }
-    />
-    <PropertyItem label="Provider" value="Red Hat, Inc" />
-    <PropertyItem label="Health Index" value="A" />
-    <PropertyItem
-      label="Repository"
-      value={
-        <a href="https://quay.io/repository/redhat/prometheus-operator">
-          https://quay.io/repository/redhat/prometheus-operator
-        </a>
-      }
-    />
-    <PropertyItem
-      label="Container Image"
-      value={
-        <a href="#">
-          0.22.2 <ExternalLinkAltIcon />
-        </a>
-      }
-    />
-    <PropertyItem
-      label="Created At"
-      value={
-        <span>
-          <GlobeIcon /> Aug 23, 1:58pm
-        </span>
-      }
-    />
-    <PropertyItem label="Support" value={<a href="#">Red Hat</a>} />
-  </PropertiesSidePanel>
-</div>
+<PropertiesSidePanel>
+  <PropertyItem label="Operator Version" value="0.9.8 (latest)" />
+  <PropertyItem
+    label="Certified Level"
+    value={
+      <span>
+        <OkIcon style={{color: '--pf-t--global--icon--color--status--success--default'}} /> Certified
```

Comment:
updated this color token since the previous was just a generic success color and we have an icon specific success color.